doc/manual/cookbooks/finite_strain

```diff
@@ -0,0 +1 @@
+../../../cookbooks/finite_strain/doc/finite_strain/
```

Comment:
please link this to `../../../cookbooks/finite_strain` then you also dont need to duplicate the directory structure in there, but put all the documents into `cookbooks/finite_strain/doc` (instead of `cookbooks/finite_strain/doc/finite_strain`).